examples/typescript/evm/exportAccount.ts

```diff
@@ -0,0 +1,38 @@
+// Usage: pnpm tsx evm/exportAccount.ts
+
+import { CdpClient } from "@coinbase/cdp-sdk";
+import "dotenv/config";
+import { generatePrivateKey, privateKeyToAccount } from "viem/accounts";
+
+const cdp = new CdpClient();
+
+const privateKey = generatePrivateKey();
+console.log("Generated private key: ", privateKey);
```

Comment:
nit: console.log automatically adds a space in between positional args
```suggestion
console.log("Generated private key:", privateKey);
```